Xamarin.Forms.Core.UITests.Shared/Utilities/AppExtensions.cs

```diff
@@ -28,8 +28,12 @@ public static void NavigateToGallery (this IApp app, string page)
 			var text = Regex.Match (page, "'(?<text>[^']*)'").Groups["text"].Value;
 
 			app.WaitForElement("SearchBar");
-			app.EnterText (q => q.Raw ("* marked:'SearchBar'"), text);
+			app.EnterText(q => q.Raw("* marked:'SearchBar'"), text);
 
+			//var element = app.Query("SearchBar").First();
```

Comment:
Get rid of the commented code if we don't need it.

Reply:
shouldn't had commit this was testing stuff on iOS